project_generator/templates/cmakelistgccarm.tmpl

```diff
@@ -64,7 +64,7 @@ set(SOURCES_S ${SOURCES_S} "{{file}}"){% endfor %}
 {% for file in source_files_obj %}
 set(OBJECT_FILES ${OBJECT_FILES} "{{file}}"){% endfor %}
 
-set(COMMON_FLAGS "-mcpu={{core}} {% for flag in misc['common_flags'] %} {{flag}}{% endfor %}")
+set(COMMON_FLAGS "-mcpu={{core}} -mthumb {% for flag in misc['common_flags'] %} {{flag}}{% endfor %}")
```

Comment:
shouldn't this be in the common flags ? I would assume arm mode is possible, not only thumb